accessibility-oscilloscope/package/oscilloscope-trigger/oscilloscope-trigger.mk

```diff
@@ -0,0 +1,19 @@
+################################################################################
+#
+# oscilloscope-trigger
+#
+################################################################################
+
+OSCILLOSCOPE_TRIGGER_VERSION = v0.0.1
```

Comment:
this does not line up with any tags in the target repository.
also, the target repo is private ----- none of this will work unless it's publicly accessible